Src/AutoFixture/Kernel/ParameterSpecification.cs

```diff
@@ -95,7 +95,9 @@ private bool ParameterIsCompatibleWithTargetType(object request)
 
         private bool ParameterHasTargetName(object request)
         {
-            return ((ParameterInfo)request).Name == this.targetName;
+            return ((ParameterInfo)request)
+                   .Name
+                   .Equals(this.targetName, StringComparison.OrdinalIgnoreCase);
```

Comment:
See [this comment](https://github.com/AutoFixture/AutoFixture/pull/318#discussion_r26196130)